src/routes/(app)/checkout/+page.server.ts

```diff
@@ -171,7 +171,11 @@ export const actions = {
 										city: z.string().min(1),
 										state: z.string().optional(),
 										zip: z.string().min(1),
-										country: z.enum([...COUNTRY_ALPHA2S] as [CountryAlpha2, ...CountryAlpha2[]])
+										country: z.enum([...COUNTRY_ALPHA2S] as [CountryAlpha2, ...CountryAlpha2[]]),
+										phone: z
+											.string()
+											.regex(/^\d+(\.\d+)?$/)
+											.default('0')
```

Comment:
no, why would be store `0` in a phone number field? Let's not add a regex either. `maxLength` if you want.

(a regex could be `/^([+]?[\s0-9]+)?(\d{3}|[(]?[0-9]+[)])?([-]?[\s]?[0-9])+$/` but not needed IMO)

Reply:
Same thought, especially as we'll allow 0033123456789 or +33123456789 or 0123456789 or either "call 0123456789 or 9876543210 depending of the time of the day" or anything else on input for now, that's why I didn't say a thing about phone number format & input restriction in the ticket.